package.json

```diff
@@ -55,6 +55,7 @@
     "webextension-polyfill": "^0.6.0"
   },
   "dependencies": {
+    "geckodriver": "^1.21.0",
```

Comment:
This should probably be a dev dependency, not a runtime dependency, since we only need it for tests.